src/resizable-card-layout.scss

```diff
@@ -0,0 +1,137 @@
+@import "./new-settings";
+@import "./mixins";
+@import "./icon";
+
+$block: #{$fd-namespace}-resizable-card-layout;
+
+.#{$block} {
+  $fd-resizable-card-min-width: 20rem;
+  $fd-resize-handle-width: 0.0625rem;
+  $fd-border-on-drag: 0.125rem dashed var(--sapContent_DragAndDropActiveColor);
+
+  $fd-resizable-card-padding-x: (
+    s: 0.5rem,
+    m: 1rem,
+    l: 3rem,
+  );
+
+  @include fd-reset();
+
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+
+  &__wrapper {
+    @include fd-reset();
+    @include fd-flex();
+
+    min-width: $fd-resizable-card-min-width;
+    margin: 0.5rem;
+    position: relative;
+
+    &__card {
+      @include fd-reset();
+
+      order: 1;
+      flex: 1;
+      position: relative;
+      // subtract width taken by vertical handler and horizontal handler
+      height: calc(100% - #{$fd-resize-handle-width});
+
+      &__icon-wrapper {
+        @include fd-reset();
+
+        width: 1rem;
+        height: 1rem;
+        position: absolute;
+        bottom: 0;
+        right: 0;
+        z-index: 2;
+        padding: 0 0.0625rem 0.0625rem 0;
+
+        @include fd-rtl() {
+          right: auto;
+          left: 0;
+          padding: 0 0 0.0625rem 0.0625rem;
+        }
+
+        &--icon {
+          @include fd-reset();
+          @include fd-icon("resize-corner");
+
+          font-family: SAP-icons;
+          font-size: 1rem;
+          line-height: 1rem;
+          cursor: nwse-resize;
+          position: absolute;
+          bottom: 0;
+          right: 0.0625rem;
+          color: var(--sapButton_IconColor);
+
+          @include fd-rtl() {
+            transform: rotate(90deg);
+            position: relative;
+            cursor: nesw-resize;
+            padding: 0 0.0625rem 0.0625rem 0.5rem;
+            // padding-left is needed after rotation
+          }
+        }
+      }
+    }
+
+    &__resize {
+      &__cursor {
+        &--horizontal {
+          cursor: ew-resize;
+        }
+
+        &--vertical {
+          cursor: ns-resize;
+        }
+
+        &--both {
+          cursor: nwse-resize;
+
+          @include fd-rtl() {
+            cursor: nesw-resize;
+          }
+        }
+      }
+
+      &__handle {
+        &--vertical {
+          order: 2;
+          // subtract resize-corner icon font-size/2
+          width: $fd-resize-handle-width;
+          cursor: ew-resize;
+        }
+
+        &--vertical::content {
+          content: "";
+          position: absolute;
+        }
+
+        &--horizontal {
+          // subtract resize-corner icon font-size/2
+          width: calc(100% - 0.5rem);
+          height: $fd-resize-handle-width;
+          position: absolute;
+          bottom: 0;
+          cursor: ns-resize;
+        }
+      }
+
+      &--indication-border {
+        @include fd-reset();
+
+        border: $fd-border-on-drag;
+      }
+    }
+  }
+
+  @each $size-label, $size in $fd-resizable-card-padding-x {
+    &--#{$size-label} {
```

Comment:
I think we are following size names as `sm`, `md`, `lg` etc. now rather than `s`, `m`, `l`.

Reply:
fixed.